subsys/bluetooth/host/hci_core.c

```diff
@@ -407,6 +407,14 @@ static int hci_le_read_max_data_len(uint16_t *tx_octets, uint16_t *tx_time)
 	*tx_time = sys_le16_to_cpu(rp->max_tx_time);
 	net_buf_unref(rsp);
 
+	/* All limits according to BT Core spec 5.4 [Vol 4, Part E, 7.8.46] */
+	if (0x001B > *tx_octets || *tx_octets > 0x00FB) {
+		LOG_WRN("tx_octets exceeds the valid range %d", *tx_octets);
+	}
+	if (0x0148 > *tx_time || *tx_time > 0x4290) {
+		LOG_WRN("tx_time exceeds the valid range %d", *tx_octets);
```

Comment:
If the maximum values are defined in `hci_types.h`, then please use those defines. 

If they are not defined in there, please defined them and use them

Reply:
Done. Could you check it again?